bootstrap/Dockerfile

```diff
@@ -34,6 +34,10 @@ RUN git clone https://github.com/ksonnet/ksonnet.git /opt/ksonnet && \
 	cd /opt/ksonnet && \
 	git checkout v0.10.0-alpha.3
 
+RUN git clone https://github.com/kubeflow/kubeflow.git /opt/kubeflow && \
```

Comment:
do git clone --depth 1 - we don't need whole git history (couple hundreds of megs)